node/src/chain/acceptor.rs

```diff
@@ -249,6 +265,8 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
             state_hash = to_str(&blk.header().state_hash),
             fsv_bitset,
             ssv_bitset,
+            block_time,
+            dur = format!("{:?}ms", duration.as_millis()),
```

Comment:
no need for debug format here
```suggestion
            dur = format!("{}ms", duration.as_millis()),
```

Reply:
In order to be more usable by metrics, it would be better to completely remove the `ms` suffix, and specify it in the name like `dur_ms`

Reply:
Updated